### DISCUSSION TITLE
Completely filled in personal information but still received a prompt to add personal information

### DISCUSSION THREAD

**Discussion by TinyuZhao** [community/community#52376] [GitHub Education]
Why do I keep getting the "completed your GitHub user profile" message even though I've filled out almost all of my profile?

<img width="373" alt="image" src="https://user-images.githubusercontent.com/44396641/230781365-3eb09445-96ed-4ce9-80de-871ab2e81c3f.png">

<img width="816" alt="SCR-20230409-udsb" src="https://user-images.githubusercontent.com/44396641/230781454-576e945a-8c14-4a8f-a706-2cb8d0beb9ac.png">

**Comment by jamesopppp** [comment]
+1

**Comment by samus-aran** [comment]
Hi @Tinyu-Zhao 

Often this common pre-processing warning is displayed because the application detects differences between your profile information and the evidence you’ve supplied. We do request that students can demonstrate that their GitHub account corresponds with the evidence given. The verification process looks at your GitHub username, email, and name (as well as other points) to cross-reference the evidence submitted, so be sure that they match to ensure a smoother process.

We would also suggest that when you update your profile information that you log out and log back into GitHub before re-applying.

I would also suggest that you ensure you select the most appropriate **evidence type** in the dropdown. The order of options indicates the preference of evidence with Academic Email being at the top. If you don't have an Academic email or Student ID, do try and acquire evidence that is positioned higher rather than lower in that option list.

If you have been repetitively unsuccessful using a particular evidence type (e.g. Student ID) we would recommend that you acquire and submit an alternative piece of evidence that can verify your student status.

Let us know how you get on 👍

**Reply by TinyuZhao** [reply]
> Hi @Tinyu-Zhao
> 
> Often this common pre-processing warning is displayed because the application detects differences between your profile information and the evidence you’ve supplied. We do request that students can demonstrate that their GitHub account corresponds with the evidence given. The verification process looks at your GitHub username, email, and name (as well as other points) to cross-reference the evidence submitted, so be sure that they match to ensure a smoother process.
> 
> We would also suggest that when you update your profile information that you log out and log back into GitHub before re-applying.
> 
> I would also suggest that you ensure you select the most appropriate **evidence type** in the dropdown. The order of options indicates the preference of evidence with Academic Email being at the top. If you don't have an Academic email or Student ID, do try and acquire evidence that is positioned higher rather than lower in that option list.
> 
> If you have been repetitively unsuccessful using a particular evidence type (e.g. Student ID) we would recommend that you acquire and submit an alternative piece of evidence that can verify your student status.
> 
> Let us know how you get on 👍

I tried transcripts, student IDs, both in Chinese and English, but all to no avail, and tried a dozen times to change the information.

**Reply by samus-aran** [reply]
Ok, let's stick to English first. Update your profile information to the English equivalent. Log out and in again to GitHub. Translate your pieces of evidence ensuring the information matches key details in your profile e.g. your english name. Please try and re-submit after this has been done.

Let us know how you get on 👍
